### PR TITLE
4886 GroupEmailDespatch

### PR DESCRIPTION
**Pull request by sophiemachin**
* views
  * `userlist` and `userlistmembership`
  * `aggregated_user_list` - id here is group_id or project_task_id, for example
  * `aggregated_user_list_membership`  id here is same as above - is this ok?
* `userlist` and `userlistmembership` sample data

### CODE REVIEW

**Comment by this-andy** [comment]
https://this.tpondemand.com/entity/4886

File: api/local/database-view-sql/view_user_list_for_task_completion_membership.sql [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
 CREATE OR REPLACE VIEW public.user_list_for_task_completion_membership AS
     SELECT
-        ulfc.project_task_id,
+        ulfc.id,
         up.anon_project_specific_user_id
```
Comment (this-andy) [inline api/local/database-view-sql/view_user_list_for_task_completion_membership.sql:22]: Do we not need to add project id in here too?  See also comment below

File: api/local/database-view-sql/view_user_list_for_user_group_membership.sql [hunk 2]
```diff
@@ -18,9 +18,9 @@
 
 CREATE OR REPLACE VIEW public.user_list_for_user_group_membership AS
     SELECT
-        group_id,
+        ulfug.id,
         ulfug.project_id,
```
Comment (this-andy) [inline api/local/database-view-sql/view_user_list_for_user_group_membership.sql:22]: ...or remove this project id.  The membership views should all return the same columns I think

Comment (sophiemachin) [reply api/local/database-view-sql/view_user_list_for_user_group_membership.sql:22]: The usergroupmembership table doesnt have project_id so I think I'll remove it

**Review by this-andy — APPROVED** [review]

**Pull request merged**